tests/integration/test_old_models/test_old_submodels/test_old_interface.py

```diff
@@ -66,9 +66,9 @@ def test_simplify_constant_current(self):
 
         j = disc.process_symbol(parameter_values.process_symbol(j))
 
-        # Simplifiy, since current is constant this should give a vector
+        # Simplifiy, since current is constant this should give an array when evaluated
         j_simp = j.simplify()
-        self.assertIsInstance(j_simp, pybamm.Vector)
+        self.assertIsInstance(j_simp.evaluate(t=0), np.ndarray)
```

Comment:
This test no longer really does anything, as j will always give an array when evaluated. Probably best to remove, or think about how the `GetConstantCurrent` function would simplify

Reply:
good spot!